packages/shared/src/components/squads/Members/PrivilegedMemberItem.tsx

```diff
@@ -17,14 +18,16 @@ export function PrivilegedMemberItem({
     <ProfileTooltip userId={user.id} tooltip={{ placement: 'bottom' }}>
       <ProfileLink
         href={user.permalink}
-        className="flex flex-row items-center rounded-10 border border-border-subtlest-tertiary p-2"
+        className="flex flex-row items-center gap-2.5 rounded-10 border border-border-subtlest-tertiary p-2"
       >
         <ProfilePicture user={user} size={ProfileImageSize.Large} />
         <div className="flex-col">
-          <span className="ml-2.5 flex truncate text-text-tertiary typo-subhead">
+          <span className="flex truncate text-text-tertiary typo-subhead">
             {user.name}
           </span>
-          <SquadMemberBadge role={role} />
+          <UserBadge className="w-fit" role={role}>
```

Comment:
Maybe have some vertical padding between these? Feels a bit cramped, like the badge is touching the text
![image](https://github.com/user-attachments/assets/e893afb0-084a-42f7-bbeb-534b2af782f8)
![image](https://github.com/user-attachments/assets/c2bb24a4-e198-43ff-bc73-8fca4132f29d)